modules/shortcodes/slideshare.php

```diff
@@ -111,5 +111,10 @@ function slideshare_shortcode( $atts ) {
 
 	$player .= ' allowfullscreen webkitallowfullscreen mozallowfullscreen></iframe>';
 
-	return $player;
+	/**
+	 * Filter the returned shortcode.
+	 *
```

Comment:
My plan was to add those in in a subsequent change -- I'd been thinking PR, but maybe it would be easier to just do this all in this PR and then resync to wpcom.